app/kuma-cp/cmd/cmd_suite_integration_test.go

```diff
@@ -0,0 +1,52 @@
+// +build k8s,integration
+
+package cmd_test
+
+import (
+	"testing"
+
+	"github.com/Kong/kuma/tools/test/framework"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestIntegrationCmd(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Integration CMD Suite")
+}
+
+var _ = Describe("Test K8s deployment with `kumactl install control-plane`", func() {
+
+	It("Deploy on Single K8s cluster and verify the Kuma CP REST API is accessible", func() {
+		t := framework.NewK8sTest(1, "", framework.Silent)
+
+		err := t.DeployKumaOnK8sClusterE(1)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = t.VerifyKumaOnK8sClusterE(1)
+		Expect(err).ToNot(HaveOccurred())
+
+		_ = t.DeleteKumaOnK8sClusterE(1)
+		_ = t.DeleteKumaNamespaceOnK8sClusterE(1)
+	}, 90)
```

Comment:
How about slightly different approach. Let's name the clusters and do it like this
```
type Clusters interface {
    Get(name string) Cluster
    Cluster
}

type Cluster interface {
    DeployKuma()
    VerifyKuma()
    DeleteKuma()
}

It("Deploy on Single K8s cluster and verify the Kuma CP REST API is accessible", func(done Done) {
    clusters := framework.NewK8sTest(framework.Silent)

    err := clusters.Get("one").DeployKuma()
    Expect(err).ToNot(HaveOccurred())

    err := clusters.Get("one").VerifyKuma()
    Expect(err).ToNot(HaveOccurred())

    err := clusters.Get("one").DeleteKuma()

    close(done)
}, 90)
```